modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -456,3 +452,7 @@ $ oc patch Network.operator.openshift.io cluster --type='merge' \
 ----
 $ oc delete namespace openshift-sdn
 ----
+
+.Next steps
+
+* Optional: After cluster migration, you can convert your IPv4 single-stack cluster to a dual-network cluster network that supports IPv4 and IPv6 address families. For more information, see "Converting to IPv4/IPv6 dual-stack networking".
```

Comment:
Just a thought - Consider adding an Additional resources section after this module with a link to "Converting to IPv4/IPv6 dual-stack networking".